websub-native/src/main/java/org/ballerinalang/net/websub/nativeimpl/HubNativeOperationHandler.java

```diff
@@ -60,9 +58,13 @@
      * @param webSubHub respective hub instance
      * @return an array of available topics
      */
-    public static ArrayValue getAvailableTopics(BObject webSubHub) {
+    public static BArray getAvailableTopics(BObject webSubHub) {
         String[] topics = Hub.getInstance().getTopics();
-        return new ArrayValueImpl(StringUtils.fromStringArray(topics));
```

Comment:
we can directly return StringUtils.fromStringArray(topics) which provides BArray